pkg/profiler/cpu/maps.go

```diff
@@ -44,16 +44,23 @@ const (
 	maxUnwindSize         = maxUnwindTableSize * unwindTableShardCount
 )
 
-type BPFCFAType uint16
+type BpfCfaType uint16
 
 const (
-	CFARegisterUndefined BPFCFAType = iota
-	CFARegisterRbp
-	CFARegisterRsp
-	CFARegisterExpression
+	CfaRegisterUndefined BpfCfaType = iota
+	CfaRegisterRbp
+	CfaRegisterRsp
+	CfaRegisterExpression
 )
 
-const CFAFakeOffset = int16(0xCFA)
+type BPFRbpType uint16
+
+const (
+	RBPRuleOffsetUnchanged BPFRbpType = iota
```

Comment:
```suggestion
	RbpRuleOffsetUnchanged BPFRbpType = iota
```
If we want to be consistent with your other naming changes. I know it's hard with abbreviations. Let's at least be consistent.

Reply:
Ohh true, will fix 😄